bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/EnergyInfoDayAndWeek.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.daikin.internal.api;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Holds information from the get_week_power_ex call.
+ *
+ * @author Wouter Denayer <wouter@denayer.com> - Added to support for weekly & daily energy reading
+ *
+ */
+@NonNullByDefault
+public class EnergyInfoDayAndWeek {
+    public Optional<Double> energyHeatingToday = Optional.empty();
+    public Optional<Double> energyHeatingThisWeek = Optional.empty();
+    public Optional<Double> energyHeatingLastWeek = Optional.empty();
+    public Optional<Double> energyCoolingToday = Optional.empty();
+    public Optional<Double> energyCoolingThisWeek = Optional.empty();
+    public Optional<Double> energyCoolingLastWeek = Optional.empty();
+
+    private static final Logger logger = LoggerFactory.getLogger(EnergyInfoDayAndWeek.class);
+
+    private EnergyInfoDayAndWeek() {
+    }
+
+    public static EnergyInfoDayAndWeek parse(String response) {
+
+        EnergyInfoDayAndWeek info = new EnergyInfoDayAndWeek();
+
+        logger.debug("Parsing string: \"{}\"", response);
```

Comment:
Can this message be replaced by using the debugger or by increasing the framework's log level? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
agree, this should be trace level probably. Will change it in EnergyInfoYear as well.

Reply:
It seems to me that this output is only needed during development. Can you use the debugger if you want to inspect it?

Reply:
well.. I can't... haven't worked with the debugger. I guess it should be possible. It's only one line, so why not keep it in trace?